src/components/snapshot/DataAccessControlGroup.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import { Typography, Grid } from '@mui/material';
+import { TdrState } from '../../reducers';
+import UserList from '../UserList';
+
+type DataAccessControlGroupProps = {
+  authDomains: Array<string>;
+};
+
+function DataAccessControlGroup(props: DataAccessControlGroupProps) {
+  const { authDomains } = props;
+  return (
+    <Grid container spacing={1} data-cy="data-access-control-container">
+      <Typography variant="h6">Data Access Control Groups</Typography>
+      <Grid item xs={12} data-cy="data-access-controls">
```

Comment:
Could we also include subtext that matches the description given in workspaces?  "Collaborators must be a member of all of these [groups](https://bvdp-saturn-dev.appspot.com/#groups) to access this snapshot." ideally with a link to the corresponding Terra environment's groups page.

<img width="480" alt="Screenshot 2024-01-02 at 10 37 47 AM" src="https://github.com/DataBiosphere/jade-data-repo-ui/assets/79769153/39fe875f-e07a-4a5e-a64e-10e745c07cc0">

Often we include details like this when hovering on an info icon, but my preference would be to fully render it as is done in workspaces.

Reply:
Added!